gammapy/image/utils.py

```diff
@@ -375,6 +376,47 @@ def coordinates(image, world=True, lon_sym=True, radians=False, system=None):
     return lon, lat
 
 
+def detect_peaks(image, threshold, mask=None, world=True):
```

Comment:
Maybe wait with this PR until the photutils version is merged in `photutils` and then use it here?